style/workLoggingSystem.css

```diff
@@ -70,56 +70,84 @@ button:hover {
 }
 
 .jobTicketCard {
-    background-color: #e65100;
+    background-color: #2196f3 ;
     padding-top: 1%;
-    padding-bottom: 2%;
+    padding-bottom: 1%;
     border-radius: 5px;
     cursor: pointer;
     color: white;
 }
 
-.jobTicketCard:hover {
-    background-color: #69f0ae;
+/*.jobTicketCard:hover {
+    background-color: #00e5ff;
 }
+*/
 
 .frameTitle {
-    background-color: #ffc107;
+   
     font-weight: 300;
 }
 
-.frameContent {
-    padding-top: 2%;
+.menuTicket {
+    background-color: #263238;
+    color: white;
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
+    font-weight: 300;
+    text-align: center;
+    cursor: pointer;
+    font-family: 'Bree Serif', serif;
+    font-size: 16px;
+    margin-left: 2%;
```

Comment:
Please Remove margin and use padding to solve this problem

Reply:
okay, I will try.